platform/files.go

```diff
@@ -21,6 +21,7 @@ type AnalyzedMetadata struct {
 	PreviousImage *ImageIdentifier `toml:"image"`
 	Metadata      LayersMetadata   `toml:"metadata"`
 	RunImage      *ImageIdentifier `toml:"run-image,omitempty"`
+	BuildImage    *ImageIdentifier `toml:"build-image,omitempty"`
```

Comment:
In the spec PR for this phase, I don't see the analyzed.toml data format updated to include `build-image`

Reply:
Updated spec PR!